content/docs/guides/10_angular_relay.md

```diff
@@ -0,0 +1,369 @@
+  "peer-id",
+  "buffer",
+  "crypto",
+  "ecies-geth",
+  "secp256k1",
+  "libp2p-gossipsub",
+  "it-concat",
+  "protons"
+]
+```
+
+## Types
+
+There are some type definitions we need to install and some that we don't have.
+
+```shell
+yarn add @types/bl protons
+```
+
+Create a new folder under `src` named `@types` with the following structure:
+
+```
+src/@types
+├── protons
+│   └── types.d.ts
+└── time-cache
+    └── types.d.ts
+```
+
+In the `protons/types.d.ts` file add:
+
+```js
+declare module 'protons';
+```
+
+In the `time-cache/types.d.ts` file add:
+
+```js
+declare module "time-cache" {
+
+  interface TimeCacheInterface {
+    put(key: string, value: any, validity: number): void;
+    get(key: string): any;  
+    has(key: string): boolean;
+  }
+
+  type TimeCache = TimeCacheInterface;
+
+  function TimeCache(options: object): TimeCache;
+
+  export = TimeCache;
+
+}
+```
+
+## js-waku
+
+Then, install [js-waku](https://npmjs.com/package/js-waku):
+
+```shell
+yarn add js-waku
+```
+
+Start the dev server and open the dApp in your browser:
+
+```shell
+yarn run start
+```
+
+# Create Waku Instance
+
+In order to interact with the Waku network, you first need a Waku instance.
+Go to `app.component.ts` and add the following import:
+
+```js
+import { Waku, WakuMessage } from 'js-waku';
+```
+
+
+replace the `AppComponent` class with the following:
+
+```js
+export class AppComponent {
+  
+  title: string = 'relay-angular-chat';
+  waku!: Waku;
+  wakuStatus: string = 'None';
+
+  ngOnInit(): void {
+    Waku.create({ bootstrap: { default: true } }).then((waku) => {
+      this.wakuStatus = 'Connecting...'
+    });
+  }
+}
+
+```
+
+Add the following HTML to the `app.component.html` to show the title and render the connection status:
+
+```html
+<h1>{{title}}</h1>
+<p>Waku node's status: {{ wakuStatus }}</p>
+```
+
+# Wait to be connected
+
+When using the `bootstrap` option, it may take some time to connect to other peers.
+To ensure that you have relay peers available to send and receive messages,
+use the `Waku.waitForRemotePeer()` async function. Add the following to the 
+`Waku.create()` function:
+
+```js
+waku.waitForRemotePeer().then(() => {
+  this.wakuStatus = 'Connected';
+  this.waku = waku;
+});
+```
+
+# Define Message Format
+
+We've already installed [protons](https://www.npmjs.com/package/protons).
+Let's use it to define the Protobuf message format with two fields: 
+`timestamp` and `text`:
+
+```js
+import protons from 'protons';
+
+const proto = protons(`
+message SimpleChatMessage {
+  uint64 timestamp = 1;
+  string text = 2;
+}
+`);
+
+```
+
+Let's also define an `interface` for use with TypeScript.
+
+
+```js
+interface MessageInterface {
+  timestamp: Date,
+  text: string
+}
+```
+
+# Send Messages
+
+In order to send a message, we need to define a few things.
+
+The `contentTopic` is the topic we want subscribe to and
+the `payload` is the message. We've also defined a `timestamp`
+so let's create that.
```

Comment:
```suggestion
The `contentTopic` is the topic we want subscribe to and the `payload` is the message.
We've also defined a `timestamp` so let's create that.
```

Use sembr.

It's also a good opportunity to point to http://localhost:1313/docs/guides/01_choose_content_topic/

Reply:
There's a link to content topics in the opening paragraphs